src/hotspot/cpu/aarch64/aarch64_vector.ad

```diff
@@ -5522,12 +5522,12 @@ instruct vstoremask_truecount_neon(iRegINoSp dst, vReg src, immI_gt_1 size, vReg
     // Input "src" is a vector mask represented as lanes with
     // 0/-1 as element values.
     uint esize = (uint)$size$$constant;
-    if (esize == 8) {
-      __ addpd($vtmp$$FloatRegister, $src$$FloatRegister);
+    uint length_in_bytes = Matcher::vector_length_in_bytes(this, $src);
+    Assembler::SIMD_Arrangement arrangement = Assembler::esize2arrangement(esize,
+                                                                           /* isQ */ length_in_bytes == 16);
+    if (arrangement == __ T2D || arrangement == __ T2S) {
```

Comment:
I see `jdk/incubator/vector/Float64VectorTests.java` covers the case `arrangement == __ T2S`.
Is there a test covering the case `arrangement == __ T2D`?

Reply:
> I see `jdk/incubator/vector/Float64VectorTests.java` covers the case `arrangement == __ T2S`. Is there a test covering the case `arrangement == __ T2D`?

Thanks for your review.
I don't find such a case. I will add a test case in TestVectorMaskTrueCount.java to cover `arrangement == __ T2D`.